src/credentials/FabricTable.h

```diff
@@ -208,6 +208,9 @@ class DLL_EXPORT FabricInfo
 
     friend class FabricTable;
 
+    // Test-only, build a fabric using given root cert and NOC
+    CHIP_ERROR BuildFabric(ByteSpan rootCert, ByteSpan icacCert, ByteSpan nocCert, ByteSpan nodePubKey, ByteSpan nodePrivateKey);
```

Comment:
Lets prefix this if it is test only, so we have `TestOnly_BuildFabric` to make it clear that this is not external api.